website/docs/content/writing-guidelines/index.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Writing Guidelines
+caption: Guidelines for writing consistent UI copy.
+description: Guidelines for writing consistent UI copy.
+previewImage: assets/illustrations/content/writing-guidelines.jpg
+---
+
+## Date and time
+
+Present dates and times in a clear and consistent format to enhance usability across products.
+
+### Usage
+
+Follow these formatting options to display dates and times in a way that aligns with user needs: 
+- **Relative date**: Displays time in relative terms (e.g., "3 hours ago" or "in 2 days"). This format is recommended for recent events, particularly within the past or upcoming week, as it simplifies recognition of time-sensitive actions.
```

Comment:
```suggestion
- **Relative date**: Displays time in relative terms, e.g., "3 hours ago" or "in 2 days". This format is recommended for recent events, particularly within the past or upcoming week, as it simplifies recognition of time-sensitive actions.
```
[reference](https://hashicorp.atlassian.net/wiki/spaces/HDS/pages/3342009417/Writing+Style#Formatting-e.g.-and-i.e.)